kollider-price/src/convert.rs

```diff
@@ -0,0 +1,23 @@
+use super::price_feed::error::KolliderPriceFeedError;
+use super::price_feed::KolliderPriceTicker;
+
+use shared::{
+    payload::{
+        ExchangeIdRaw, InstrumentIdRaw, KolliderBtcUsdSwapPricePayload, PriceMessagePayload,
+        PriceRatioRaw,
+    },
+    time::TimeStamp,
+};
+
+impl TryFrom<KolliderPriceTicker> for KolliderBtcUsdSwapPricePayload {
+    type Error = KolliderPriceFeedError;
+    fn try_from(value: KolliderPriceTicker) -> Result<Self, Self::Error> {
+        Ok(KolliderBtcUsdSwapPricePayload(PriceMessagePayload {
+            exchange: ExchangeIdRaw::from("Kollider"), // FIXME
+            instrument_id: InstrumentIdRaw::from("BTC-USD-SWAP"), // FIXME "BTC-USD-SWAP"
+            timestamp: TimeStamp::now(),               //FIXME
```

Comment:
This is not great but probably fine if kollider doesn't publish a timestamp (it should!).
More accurate would be to create the timestamp in the code that received the message and then calls `KolliderBtcUsdSwapPricePayload::try_from((tick, timestamp))`